### PR TITLE
addition of new language: Mongolia

### PR DESCRIPTION
**Pull request by bayartsogt-ya**
I am adding the new language templates for the hosted API.
Please review this for me. Thank you so much!

### CODE REVIEW

File: DefaultWidget.ts [hunk 1]
```diff
@@ -361,6 +361,48 @@ const MAPPING_BN: PerLanguageMapping = new Map([
...
+			text: `Миний нэрийг хэн гэдэг вэ?`,
+			context: `Намайг Ганбат гэдэг. Би Увс аймагт төрсөн.`,
+		}
+	] ],
+	[ "translation", [
+		`Намайг Дорж гэдэг. Би Улаанбаатарт амьдардаг.`,
+		`Намайг Ганбат гэдэг. Би Увс аймагт төрсөн.`,
+	] ],
+	[ "summarization", [
+		`Mongolia (/mɒnˈɡoʊliə/ (About this soundlisten), Mongolian: Монгол Улс, transcription: Mongol Uls, Traditional Mongolian: ᠮᠤᠩᠭᠤᠯ ᠤᠯᠤᠰ, transliteration: Mongγol ulus) is a landlocked country in East Asia. Its area is roughly equivalent with the historical territory of Outer Mongolia, and that term is sometimes used to refer to the current state. It is sandwiched between Russia to the north and China to the south, where it neighbours the Inner Mongolia Autonomous Region. Mongolia does not share a border with Kazakhstan, although only 37 kilometres (23 mi) separate them.`,
```
Comment (julien-c) [inline DefaultWidget.ts]: this is in EN, is this intended?

**Review by julien-c — APPROVED** [review]
Thanks! Also cc'ing @mapmeld for info

File: DefaultWidget.ts (hunk 1 above)
Comment (bayartsogt-ya) [reply DefaultWidget.ts]: @julien-c 

Sorry for the bad validation.
I just corrected the initialization (which was in English, now in Mongolian)

Thank you for your help!

**Comment by julien-c** [comment]
Deployed: see e.g. https://huggingface.co/bayartsogt/bert-base-mongolian-cased

**Comment by bayartsogt-ya** [comment]
> Deployed: see e.g. https://huggingface.co/bayartsogt/bert-base-mongolian-cased

@julien-c 
Thank you so much

**Pull request merged**